### PR TITLE
docs: clarify that sessions use cache by default

### PR DESCRIPTION
**Pull request by nikwen**
#### Description of Change

Relevant code: https://github.com/electron/electron/blob/48801be8e9e3b898635e55abb0b596c83bb9c17a/shell/browser/electron_browser_context.cc#L332-L335

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation, tutorials, templates and examples are changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Review by ckerr — CHANGES_REQUESTED** [review]

File: docs/api/session.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ To create a `Session` with `options`, you have to ensure the `Session` with the
 `partition` has never been used before. There is no way to change the `options`
 of an existing `Session` object.
 
+By default, the `cache` option is set to `true`.
```
[the remark this answers is not in the document]
Comment (ckerr) [reply docs/api/session.md:45]: Maybe we should even link to docs/api/command-line-switches#--disable-http-cache e.g.

- \`Cache\` boolean - Whether to enable cache. Default is \`true\` unless \[the --disable-http-cache switch\]\(api/command-line-switches#--disable-http-cache\) is used.

[the remark this answers is not in the document]
Comment (nikwen) [reply docs/api/session.md]: That's a great suggestion, thanks! Updated the PR.

**Comment by nikwen** [comment]
Thanks for the review, @ckerr!

**Review by ckerr — APPROVED** [review]
Thanks for the changes!

**Comment by nikwen** [comment]
Thanks for merging!

**Pull request merged**